Automattic/jetpack
# Backup: Fix className prop on loading state

## Proposed changes:

* Replaces `class` with `className`

Currently the DevTools is throwing the following warning in local:
<img width="1007" alt="CleanShot 2023-07-24 at 08 50 24@2x" src="https://github.com/Automattic/jetpack/assets/1488641/166c5178-7786-4dcc-95b2-f1b37632faca">

This PR aims to remove this warning.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:





* Checkout this branch locally
* Navigate to the Jetpack VaultPress Backup plugin admin page and ensure no warning is being thrown on the developer console.
